src/d3a/d3a_core/redis_connections/redis_area_market_communicator.py

```diff
@@ -97,11 +97,15 @@ def publish_json(self, channel, data):
 class ExternalConnectionCommunicator(ResettableCommunicator):
     def __init__(self, is_enabled):
         self.is_enabled = is_enabled
+        self.aggregator = None
         if self.is_enabled:
             super().__init__()
             self.channel_callback_dict = {}
             self.aggregator = AggregatorHandler(self.redis_db)
 
+    def is_aggregator_controlling_device(self, device_uuid):
+        return self.aggregator and self.aggregator.is_controlling_device(device_uuid)
```

Comment:
It looks like this method is never called. Is it a leftover?

Reply:
Remnant after the merge from master, removing